packages/vm/tests/BlockchainTestsRunner.js

```diff
@@ -8,6 +8,12 @@ const level = require('level')
 const levelMem = require('level-mem')
 
 module.exports = async function runBlockchainTest(options, testData, t) {
+  // ensure that the test data is the right fork data
+  if (testData.network != options.forkConfigTestSuite) {
+    t.comment("skipping test: no data available for " + options.forkConfigTestSuite)
+    return
+  }
+
```

Comment:
This is too late in the process, false forks should be already filtered out when the test files are read (currently the code logic in `ethereumjs-util`) and there should be no output on this, this is otherwise causing too much unnecessary noise.

Will do a follow-up comment here: https://github.com/ethereumjs/ethereumjs-vm/pull/808

Reply:
I see where you are getting from, but this is not the case for [state tests](https://github.com/ethereumjs/ethereumjs-vm/blob/071fdd9a37ef24c999fc096ba387c9f1e6326c21/packages/vm/tests/GeneralStateTestsRunner.js#L8) either: files are read and are then filtered out (this is thus not done in the `ethereumjs-testing` module, but actually in the state test runner which we define in the VM package).

So this is inconsistent. Do you think we should move this check to `ethereumjs-testing`? (Such that the runner *never* skips any tests?). I see in #808 you have moved all the testing logic in the VM package and I think this is a good idea; we can then move these fork checks to a more 'natural' place (i.e. the tests filter) and then let the runner only do anything if it actually has the right data to test.

Reply:
Ok, will see this as resolved for now.